src/main/31-cloudfront.tf

```diff
@@ -72,21 +77,22 @@ module "fe_cdn" {
   }
   ordered_cache_behavior = [
     {
-      path_pattern             = "/eservices/*"
+      path_pattern             = "/eservices"
```

Comment:
This will work only for the exact match `/eservices`. 
Paths like `/eservices/<eserviceId>` won't get routed towards the APIGW.

Reply:
seems like is the only path pattern used for this API

Reply:
Ok, remember to update this pattern if the developers add subpaths